.github/workflows/shippable_builds.yml

```diff
@@ -199,17 +199,36 @@ jobs:
           name: signed-${{ env.APP_NAME }}-${{ env.PACKAGE_FORMAT }}-${{ env.PACKAGE_FLAVOR }}
           path: ${{ env.UPLOADS }}/
 
-      - name: Get Tag Name
+
+      - name: Get APK Info
+        id: apkinfo
+        shell: bash
+        run: |
+          APK_FILE="${APP_NAME}-${PACKAGE_FLAVOR}-${RELEASE_TYPE}-signed.apk"
+          LATEST_BUILD_TOOLS=$(ls -d ${ANDROID_SDK_ROOT}/build-tools/* | sort -V | tail -n1)
+          AAPT=${LATEST_BUILD_TOOLS}/aapt
+
+          NAME=$(${AAPT} dump badging $APK_FILE | sed -n "s/.*application-label:'\([^']*\)'.*/\1/p")
+          VERSION=$(${AAPT} dump badging $APK_FILE | sed -n "s/.*versionName='\([^']*\)'.*/\1/p")
+
+          echo "TAG_NAME=${TAG_PREFIX}_${VERSION//./_}" >> $GITHUB_OUTPUT
+          echo "FULL_VERSION_NAME=${NAME} ${VERSION}" >> $GITHUB_OUTPUT
+          echo "VERSION_NAME=${VERSION}" >> $GITHUB_OUTPUT
+
+          cat $GITHUB_OUTPUT
+
+      - name: Rename release asset
+        id: rename
         shell: bash
+        env:
+          VERSION_NAME=${{ steps.apkinfo.outputs.VERSION_NAME }}
         run: |
-          APKANALYZER="${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/apkanalyzer"
           APK_FILE="${APP_NAME}-${PACKAGE_FLAVOR}-${RELEASE_TYPE}-signed.apk"
-          _version=$(${APKANALYZER} manifest version-name "${UPLOADS}/${APK_FILE}")
-          _tag="${TAG_PREFIX}_${_version//./_}"
-          echo "Tag Name: ${_tag}"
-          echo "Apk File: ${APK_FILE}"
-          echo "TAG_NAME=${_tag}" >> $GITHUB_ENV
-          echo "APK_FILE=${APK_FILE}" >> $GITHUB_ENV
+          APK_FILE_PRETTY="${APP_NAME/app-/}-${VERSION_NAME}.apk"
```

Comment:
@jfx2006 we might want to consider just making APP_NAME "thunderbird" and "k9mail", and then adding the `app-` prefix during the build step. This seems a bit cleaner because `app-` is really just an Android/gradle detail.